### DISCUSSION TITLE
4.4.171 [Resolved]

### DISCUSSION THREAD

**Discussion by JYewdall** [Open-Shell/Open-Shell-Menu#1093] [General]
Open-Shell update shows there is an update available . 4.4.171 - Fix corrupted icons in second column (#1088)
The download is always 1KB and not executable

**Comment by Ibuprophen** [comment]
Thanks for mentioning this. I'll see what I (or another Developer) can do. 

The reason your experiencing this is because there's a Download Limit on test builds that was reached a while ago. 

~Ibuprophen

**Comment by Ibuprophen** [comment]
@JYewdall, try the following test build I had rebuilt from the 4.4.171 changes. 

https://ci.appveyor.com/project/Ibuprophen/open-shell-menu-6wu0h/build/artifacts

~Ibuprophen

**Comment by blackcrack** [comment]
there is a zip with both Files, for download here ..:
```
Fix corrupted icons in second column #1088 
 Merged
Ibuprophen merged 1 commit into Open-Shell:master from 
among-us-official:master 2 days ago (11.08.2022)

Build Open-Shell-Menu 4.4.171-khrqiprk completed (commit e4e614b0ae by @among-us-official)
https://ci.appveyor.com/project/passionate-coder/open-shell-menu/builds/44422956

 Pull request #1088 - Fix corrupted icons in second column
Fix corrupted icons in second column
When using skin with a different second column icon size, all of the
icons in it would become blank.

Because m_bTwoColumns is set after AddStandardItems, it is still false
in the column break check. As a workaround, use the value it's assigned
later in InitWindowInternal instead.
```

info.txt + OpenShellSetup_4_4_171-khrqiprk.exe + OpenShellPDB_4_4_171-khrqiprk.7z = [4.4.171-khrqiprk.zip](https://github.com/Open-Shell/Open-Shell-Menu/files/9331958/4.4.171-khrqiprk.zip)

i guess here is no limit ;)

best

**Comment by Ibuprophen** [comment]
@ge0rdi, can you please push a new version release (based upon the recent changes made)? 

Unless you object it at this point in time? 👍 

~Ibuprophen

**Comment by JYewdall** [comment]
I used open-shell update to successfully update  to 4.4.172. Thanks.
You haven't updated https://github.com/Open-Shell/Open-Shell-Menu/releases - still showing 4.4.170 as the latest.

**Comment by blackcrack** [comment]
https://ci.appveyor.com/project/passionate-coder/open-shell-menu/branch/master/artifacts  

ehhh..
```
{"message":"Artifacts download limit (1024 MB/day) exceeded."}
```   
there we see how many peoples loves OpenShellMenu .. huhh..
@JYewdall or @Ibuprophen  could you maybe make also a package with both files and add it to your Post as attachment in this tread ?
(and rename the header to 4.4.172 ?)

best

**Comment by Ibuprophen** [comment]
@ge0rdi, @blackcrack & @JYewdall, I believe that I may have (hopefully) done it correctly this time around. 👍 

https://github.com/Open-Shell/Open-Shell-Menu/releases

I'm a bit rusty regarding a push on new releases... LOL! 

~Ibuprophen
